platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ExportMigrationContextImplTest.java

```diff
@@ -527,6 +550,31 @@ public void testEntriesWithFilterAndRelativePath() throws Exception {
     Assert.assertThat(report.hasWarnings(), Matchers.equalTo(false));
   }
 
+  @Test
+  public void testEntriesWhenNotRecursingWithFilterAndRelativePath() throws Exception {
+    final Path etc = createDirectory("etc");
+    final Path ks = createDirectory("etc", "keystores");
+    final Path other = createDirectory("other");
+    final List<Path> paths = createFiles(etc, "test.cfg", "test2.config", "test3.properties");
+
+    createFiles(ks, "testServerKeystore.jks");
+    createFiles(ks, "serverKeystore.jks", "serverTruststore.jks");
+    createFiles(other, "a", "b");
+    createDirectory("etc", "keystores", "empty");
+
+    final List<ExportMigrationEntry> entries =
+        context
+            .entries(etc, false, p -> p.getFileName().toString().startsWith("test"))
```

Comment:
❓Are we adding any other files under `etc` than those we're adding on L558? If not, shouldn't we have a file that doesn't start with `test` under `etc` to make sure the filter works?